modular_zubbers/code/modules/synths/code/synth_prefab.dm

```diff
@@ -0,0 +1,17 @@
+/mob/living/carbon/human/species/synth
+	race = /datum/species/synthetic
+
+// Used for printing dead synth bodies
+/mob/living/carbon/human/species/synth/empty
+
+/mob/living/carbon/human/species/synth/empty/Initialize(mapload)
+	. = ..()
+	var/old_death_sound
```

Comment:
this is never set to anything other than null

Reply:
Ok this was actually missed as I was making this PR on 3 different codebases with IPCs. Woops